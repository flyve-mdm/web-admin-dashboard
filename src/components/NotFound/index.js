import React from 'react'
import { I18n } from 'react-i18nify'

const NotFound = props => {

    return (
        <div className="authentication" style={{ textAlign: 'center'}} >
            <section>
                <figure>
                    <img alt="Flyve MDM Dashboard" src={require('../../assets/images/dashboard.svg')} />
                </figure>
                <h1>{I18n.t('commons.not_found')}</h1>
                <h1>404</h1>
            </section>
            <footer>
                <a href="https://flyve-mdm.com/privacy-policy/">
                    {I18n.t('commons.terms_and_conditions')}
                </a>
                <br />

                <span>
                    Â© 2017 - 2018 Teclib'.
                    </span>
                <br />
            </footer>
        </div>
    )
}

export default NotFound