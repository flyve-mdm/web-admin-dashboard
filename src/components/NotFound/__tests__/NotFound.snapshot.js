import React from 'react'
import renderer from 'react-test-renderer'

import NotFound from "../index"

describe('NotFound', () => {
    test('renders NotFound', () => {
        const component = renderer.create(
            <NotFound />
        )
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})