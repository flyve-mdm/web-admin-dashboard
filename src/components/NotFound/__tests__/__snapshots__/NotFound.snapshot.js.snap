// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders NotFound 1`] = `
<div
  className="authentication"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <section>
    <figure>
      <img
        alt="Flyve MDM Dashboard"
        src="dashboard.svg"
      />
    </figure>
    <h1>
      not_found
    </h1>
    <h1>
      404
    </h1>
  </section>
  <footer>
    <a
      href="https://flyve-mdm.com/privacy-policy/"
    >
      terms_and_conditions
    </a>
    <br />
    <span>
      Â© 2017 - 2018 Teclib'.
    </span>
    <br />
  </footer>
</div>
`;
